pkg/plugins/policies/embedworkaround/api/v1alpha1/schema.yaml

```diff
@@ -0,0 +1 @@
+# this is a workaround for go:embed not allowing to have an empty match on the pattern
```

Comment:
But it will be empty only if there are no policies in `pkg/plugins/policies`?

Reply:
yes, but during lint-ing phase they are removed and then recreated and in [build/protoc-gen-kumapolicy](https://github.com/kumahq/kuma/blob/4049e0a54ee834020496cdf3319639bc68623c96/mk/generate.mk#L74) we depend on a package that requires [resources_model](https://github.com/kumahq/kuma/blob/d07a50d9f0e12d06d464c5dc548843a0e24236c7/tools/policy-gen/protoc-gen-kumapolicy/utils.go#L10) and that requires [plugins](https://github.com/kumahq/kuma/blob/0671e71000e6b6f9f0a6fa8f37ecebd1cd413775/pkg/core/resources/model/resource.go#L77) and there we do the validation that requires the schema files. I think this [comment](https://github.com/kumahq/kuma/pull/5107#discussion_r992264264) is a better approach.

Reply:
done